pkg/rancher-desktop/utils/version.ts

```diff
@@ -0,0 +1,39 @@
+import { app } from 'electron';
+
+import { spawnFile } from '@pkg/utils/childProcess';
+
+function getProductionVersion() {
+  try {
+    return app.getVersion();
+  } catch (err) {
+    console.log(`Can't get app version: ${ err }`);
+
+    return '?';
+  }
+}
+
+async function getDevVersion() {
+  try {
+    const { stdout } = await spawnFile('git', ['describe', '--tags'], { stdio: ['ignore', 'pipe', 'inherit'] });
```

Comment:
Hmm, we should probably set `cwd` here. (Not necessary for this PR, since it's just moving code around.)

Reply:
Yeah, I'm going to leave this be for the time being for the same reason of just moving code around.